depthai_examples/common_src/ExposureParameters.cpp

```diff
@@ -0,0 +1,18 @@
+#include "common.hpp"
+
+void ExposureSettings::setExposure(dai::Device& device) {
+    auto controlQueue = device.getInputQueue("control");
+    auto configQueue = device.getInputQueue("config");
+    dai::CameraControl ctrl;
+    if(auto_exposure) {
+        ctrl.setAutoExposureEnable();
+    } else {
+        ctrl.setManualExposure(exposure_time_us, sensitivity_iso);
+        if(exposure_region.at(2) != 0 || exposure_region.at(3) != 0) {
+            dai::ImageManipConfig cfg;
+            cfg.setCropRect(exposure_region.at(0), exposure_region.at(1), exposure_region.at(2), exposure_region.at(3));
+            configQueue->send(cfg);
+        }
+    }
+    controlQueue->send(ctrl);
+}
```

Comment:
https://docs.luxonis.com/en/latest/pages/faq/?highlight=Exposure%20setting#white-balance-awb

Can you add AF, AE to the same ?

Reply:
Can we add this also in such a way that it can be changed live using dynamic reconfigure and using services?

That would be a great addition to some of the users who wants to test with different settings and stop at one which works best for them.